pages/manager/PluginHandler.inc.php

```diff
@@ -68,6 +68,21 @@ function plugins($args, &$request) {
 		$templateMgr->assign('helpTopicId', 'journal.managementPages.plugins');
 
 		$site =& $request->getSite();
+		$pluginDir = dirname(dirname(dirname(__FILE__))).'/plugins/';
+		// Don't show the install link unless something can be installed somewhere
```

Comment:
If there's a check on the specific plugin, then this check is good too. Re: "safer", I was worried that permissions might be mixed inside an installed plugin, potentially allowing the installation of some parts but failing on others.